Xamarin.Forms.Platform.iOS/ContextActionCell.cs

```diff
@@ -632,7 +632,7 @@ UIView SetupButtons(nfloat width, nfloat height)
 
 		void SetupSelection(UITableView table)
 		{
-			for (var i = 0; i < table.GestureRecognizers.Length; i++)
+			for (var i = 0; i < table.GestureRecognizers?.Length; i++)
```

Comment:
if this fixes the issue, I'm fine.

but please move the null check out of the `for` loop. This adds complexity to the inner loop that is executed every time